pym/bob/cmds/build/builder.py

```diff
@@ -318,6 +319,19 @@ def setDownloadMode(self, mode):
             assert mode == 'no'
             self.__archive.wantDownload(False)
 
+    def setDownloadLayerMode(self, modes):
+        self.__downloadLayerModes = []
+        self.__archive.wantDownload(True)
+        for mode in modes:
+            regex = ""
+            m = re.match('^(\w+)(=(.*))?$', mode)
+            modeKey = m.group(1)
+            try:
+                regex = re.compile(m.group(3))
```

Comment:
No need for a regex. A `str.partition()` is sufficient:

```suggestion
            modeKey, _, modeLayer = mode.partition("=")
            try:
                regex = re.compile(modeLayer)
```

Reply:
Changed.